modules/rules_buf/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/bufbuild/rules_buf",
+    "maintainers": [
+        {
+            "email": "alex@aspect.dev",
+            "github": "alexeagle",
+            "name": "Alex Eagle"
+        }
+    ],
+    "repository": [
+        "github:aspect-forks/rules_buf"
```

Comment:
aspect-forks?

Reply:
I decided to refactor this PR to come from bufbuild/rules_buf which means I need a fatter patch on here, rather than rely on commits that are only on aspect-forks. Greening up again.